WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeCreateCampaignIntroView.swift

```diff
@@ -0,0 +1,262 @@
+        }
+    }
+
+    var illustration: some View {
+        ZStack(alignment: .bottomLeading) {
+            Image(uiImage: UIImage.blazeIntroIllustration)
+
+            VStack(alignment: .leading, spacing: 0) {
+                Text(Localization.Illustration.startingFrom)
+                    .fontWeight(.semibold)
+                    .font(.caption2)
+                    .foregroundColor(Color(.text))
+
+                Text(Localization.Illustration.price)
+                    .fontWeight(.bold)
+                    .titleStyle()
+            }
+            .padding(Layout.Illustration.backgroundPadding)
+            .background(
+                RoundedRectangle(cornerRadius: Layout.Illustration.cornerRadius)
+                    .fill(Color(.basicBackground))
+            )
+            .padding(.bottom, Layout.Illustration.imageBottomPadding)
+        }
+    }
+}
+
+private extension BlazeCreateCampaignIntroView {
+    struct Feature {
+        let title: String
+        let subtile: String
+    }
+
+    struct FeatureView: View {
+        /// Scale of the view based on accessibility changes
+        @ScaledMetric private var scale: CGFloat = 1.0
+
+        let feature: Feature
+
+        var body: some View {
+            VStack(alignment: .leading) {
+                Image(uiImage: .blaze)
+                    .resizable()
+                    .renderingMode(.template)
+                    .foregroundColor(Color(.brand))
+                    .frame(width: Layout.logoSize * scale, height: Layout.logoSize * scale)
+
+                Text(feature.title)
+                    .fontWeight(.bold)
+                    .bodyStyle()
+
+                Text(feature.subtile)
+                    .bodyStyle()
+                    .frame(maxWidth: .infinity, alignment: .leading)
+            }
+        }
+    }
+}
+
+private extension BlazeCreateCampaignIntroView {
+    enum Localization {
+        static let title = NSLocalizedString(
+            "blazeCreateCampaignIntroView.title",
+            value: "Get your products seen by millions",
+            comment: "Title for the Blaze campaign intro view"
+        )
+        static let subtitle = NSLocalizedString(
+            "blazeCreateCampaignIntroView.subtitle",
+            value: "Our tool is designed to empower merchants with fast, simple ad campaign setups for maximum traffic boost.",
+            comment: "Subtitle for the Blaze campaign intro view"
+        )
+        enum Illustration {
+            static let startingFrom = NSLocalizedString(
+                "blazeCreateCampaignIntroView.illustration.startingFrom",
+                value: "Starting from",
+                comment: "Caption in Blaze image illustration."
+            )
+            static let price = NSLocalizedString(
+                "blazeCreateCampaignIntroView.illustration.price",
+                value: "$5",
+                comment: "Starting price in the Blaze image illustration."
+            )
```

Comment:
I don't think "$5" should be localized either. But a few things concern me here:
- When we localize this, we are unsure if it makes sense for all languages to write "start from" first and then "$5". It would be great if we could ask Ann to revise this so we can put the two texts together: "start from $5". That way, translators can have the full context and translate the phrase correctly.
- "$5" doesn't clarify whether it's USD, AUD, CAD, etc. We should also ask Ann to revise this. 
- Unless we get the price from an API, we should not put a fixed number here. 

Please feel free to add a separate GH issue to track this though.

Reply:
Those are good points. Created a separate subtask issue to track this. #11578